operator/risc_zero/risc_zero.go

```diff
@@ -0,0 +1,21 @@
+package risc_zero
+
+/*
+#cgo linux LDFLAGS: ${SRCDIR}/lib/librisc_zero_verifier_ffi.a -ldl -lrt -lm -lssl -lcrypto -Wl,--allow-multiple-definition
+#cgo darwin LDFLAGS: ${SRCDIR}/lib/librisc_zero_verifier_ffi.dylib
+
+#include "lib/risc_zero.h"
+*/
+import "C"
+import "unsafe"
+
+const (
+	MaxReceiptSize = 215523
+	MaxImageIdSize = 8
```

Comment:
as with the rust part of the ffi, there is no need to specify this constant here